core/templates/dev/head/pages/profile/profile.html

```diff
@@ -298,7 +298,6 @@ <h2 class="oppia-profile-username-small-screen">
       float: right;
       min-height: 630px;
       padding: 0 30px 30px 30px;
-      position: relative;
       width: -webkit-calc(100% - 280px);
```

Comment:
Is there any reason for this change? Any visual changes?

Reply:
In the past view <b>oppia-profile-portfolio-pages</b> defined as absolute position.and it is relative to <b>oppia-profile-content-card</b>.

but now  <b>oppia-profile-portfolio-pages</b> is not positioned as absolute.and not any other child element of <b>oppia-profile-portfolio-pages</b> is defined as absolute.that's why i removed it.
there is no change in view whether we declared it as relative or not.

Reply:
> Can you declare scope variables for the numbers on line 141.
Sure i will do.
@hoangviet1993

Reply:
That makes sense. Thanks!